templates/base.html

```diff
@@ -364,7 +372,37 @@ <h1>{{template "page_title" .}}</h1>
 
         $(document).ready(function () {
 
-                $.ajax({
+            addGlobalStyle(`
+.toast-top-right-fix {
```

Comment:
Undo this indent adjustment. Keep the beef of this PR separate from formatting changes.